### ISSUE TITLE
Error after updating ComfyUI

### ISSUE THREAD

**Issue by Neverdusk** [lilly1987/ComfyUI_node_Lilly#4]
I updated the interface recently, and now I get the following error whenever I try to open the UI:

```
Error handling request
Traceback (most recent call last):
  File "C:\Users\silve\Documents\AI Art\__Tools\_ComfyUI\python_embeded\lib\site-packages\aiohttp\web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
  File "C:\Users\silve\Documents\AI Art\__Tools\_ComfyUI\python_embeded\lib\site-packages\aiohttp\web_app.py", line 504, in _handle
    resp = await handler(request)
  File "C:\Users\silve\Documents\AI Art\__Tools\_ComfyUI\python_embeded\lib\site-packages\aiohttp\web_middlewares.py", line 117, in impl
    return await handler(request)
  File "C:\Users\silve\Documents\AI Art\__Tools\_ComfyUI\ComfyUI\server.py", line 25, in cache_control
    response: web.Response = await handler(request)
  File "C:\Users\silve\Documents\AI Art\__Tools\_ComfyUI\ComfyUI\server.py", line 153, in get_object_info
    info['input'] = obj_class.INPUT_TYPES()
  File "C:\Users\silve\Documents\AI Art\__Tools\_ComfyUI\ComfyUI\custom_nodes\Lilly Nodes\CheckpointLoaderSimpleText.py", line 26, in INPUT_TYPES
    "default": random.choice(t_checkpoints)
  File "random.py", line 378, in choice
IndexError: list index out of range

```

I'm not sure exactly what went wrong, since everything was working fine before the update. I'm still getting used to the UI, so I apologize if I missed a step somewhere.

**Comment by lilly1987** [comment]
I think your prompt words are the problem.
It would be more helpful if you show the log after updating my node.

![2023-03-27 20 00 18](https://user-images.githubusercontent.com/20321215/227923770-b6b5e197-bb8b-4402-9f2f-64ac386e7430.png)

**Comment by Neverdusk** [comment]
After reloading the UI a few times, I suddenly got it to work. I'm not sure what exactly went wrong, but I no longer receive the error. 

It may have been caused by a CheckpointLoaderSimple node I left in a previous project before closing ComfyUI. I'm not sure if I connected it to anything. But it was causing ComfyUI to fail to load entirely whenever I tried to open it. It would only display a blank page and I couldn't interact with any menu elements.

I ended up commenting out this line in ___init___.py:
`    add("CheckpointLoaderSimpleText")
`

Then I was able to load ComfyUI. The CheckpointLoaderSimple node was colored red, so I removed it. Now I can safely use the UI again. I even returned the line I commented out, and now I can add that node type again. Something may have gone wrong while updating, I assume, but it seems to work just fine now. Apologies for the false flag.
